data-prep/README.md

```diff
@@ -15,9 +15,59 @@ Data Cleaning and Munging
 
 A number of cleaning steps are necessary to use the raw data.
 
-### Project Filtering
+### Consumption Data Cleaning
+
+**Elec Traces**
+
+* Remove duplicates
```

Comment:
Even though there were not any duplicates, what fields did you look for duplicates in? (`[DIR, SPID, DATE]`)?  This should be specified so if someone is checking in the future (with a new data set) they know what to do.